public/index.css

```diff
@@ -22,6 +22,38 @@ body {
   padding: 0 3%;
 }
 
+.faq-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 25%;
+}
+
+.faq-item {
+  padding: 0 3rem;
+}
+
+/* accordion styles */
+.accordion {
+  cursor: pointer;
+  width: 100%;
+  text-align: left;
+  margin: 0.5rem 0;
+  transition: 0.4s;
```

Comment:
When we add a transition, we should have something to transition to. Right now we don't change any attribute of the `accordion` selector.
On lines 44-47, I see you changed the background color when we hover over the `accordion`. If we want to change the color smoothly on hover and not just jump to a different color, we can use a transition on the background color.
If we use a transition, it's important to define exactly what attribute we want to transition. By default, the transition works on all attributes and the browser is looking for any change. This can slow down the page and create a bad user experience. If we want to change the `background-color`, we should write:
```transition: background-color 0.4s ```

Reply:
Thanks for breaking down the transition feature. I remove it from styles for now.